lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserSwift.h

```diff
@@ -59,6 +59,12 @@ class DWARFASTParserSwift : public DWARFASTParser {
   void EnsureAllDIEsInDeclContextHaveBeenParsed(
       lldb_private::CompilerDeclContext decl_context) override {}
 
+  // FIXME: What should this do?
+  lldb_private::ConstString
+  GetDIEClassTemplateParams(const DWARFDIE &die) override {
+    return lldb_private::ConstString();
```

Comment:
Hmm, this seems like a silent time bomb.

Reply:
Yeah, likely. Not sure what this _should_ return. I plan on leaving this as-is for now and letting @adrian-prantl figure out what to do with it.

Reply:
Is this expected to get hit? Would it make sense to either have an `assert(false && "I don't know what I'm doing here");` or an `llvm_unreachable`? Seems like a nice crash somewhere would be helpful for debugging purposes if it ever does get hit.

Reply:
Yeah, a more obvious NYI would be nice (e.g. `assert(false);`).